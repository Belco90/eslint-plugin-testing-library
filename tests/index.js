'use strict';

const { configs } = require('../lib');

describe('shareable configs', () => {
  it('should export proper "recommended" config', () => {
    expect(configs.recommended).toMatchSnapshot();
  });

  it('should export proper "angular" config', () => {
    expect(configs.angular).toMatchSnapshot();
  });

  it('should export proper "react" config', () => {
    expect(configs.react).toMatchSnapshot();
  });

  it('should export proper "vue" config', () => {
    expect(configs.vue).toMatchSnapshot();
  });
});
