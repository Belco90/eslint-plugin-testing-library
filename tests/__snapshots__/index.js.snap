// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shareable configs should export proper "angular" config 1`] = `
Object {
  "plugins": Array [
    "testing-library",
  ],
  "rules": Object {
    "testing-library/await-async-query": "error",
    "testing-library/no-await-sync-query": "error",
    "testing-library/no-debug": "warn",
  },
}
`;

exports[`shareable configs should export proper "react" config 1`] = `
Object {
  "plugins": Array [
    "testing-library",
  ],
  "rules": Object {
    "testing-library/await-async-query": "error",
    "testing-library/no-await-sync-query": "error",
    "testing-library/no-debug": "warn",
  },
}
`;

exports[`shareable configs should export proper "recommended" config 1`] = `
Object {
  "plugins": Array [
    "testing-library",
  ],
  "rules": Object {
    "testing-library/await-async-query": "error",
    "testing-library/no-await-sync-query": "error",
  },
}
`;

exports[`shareable configs should export proper "vue" config 1`] = `
Object {
  "plugins": Array [
    "testing-library",
  ],
  "rules": Object {
    "testing-library/await-async-query": "error",
    "testing-library/no-await-sync-query": "error",
    "testing-library/no-debug": "warn",
  },
}
`;
